source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -292,9 +293,9 @@ package:
        and/or deserialize a property.
 
    * - ``BsonProperty``
-     - Allows for an alternative document key name when converting the POJO
-       field to BSON. Also allows a field to turn on using a discriminator
-       when storing a POJO value.
+     - Specifies a custom document field name when converting the POJO
```

Comment:
Question: Curious why "key" was changed to "field name". Any reason?

Reply:
This is for clarification. It matches the terminology we use in the rest of our MongoDB documentation. In MongoDB documents, we call them field names rather than keys (more applicable to describing JSON).